### DISCUSSION TITLE
教えてほしいです泣

### DISCUSSION THREAD

**Discussion by maruo12** [ciffelia/koe#94] [Q&A] [answered]
なぜか、サーバーに入れてしばらくたつと、コマンドが、join,leaveしか出てこなくなります

**Comment by ciffelia** [comment, marked as answer]
画像のコマンド候補にjoin, leaveしか表示されなくなるということでしょうか？

![image](https://github.com/ciffelia/koe/assets/15273128/598cb98e-c9c7-4f1f-86e4-da196f0dcbc2)

ここに表示されていないコマンドでも、キーボードで入力すれば使えるはずです。

**Reply by maruo12** [reply]
そして、ビルドのやり方が全く分からないので、参考サイトなどを教えていただきたいです
ciffeliaさんのリポジトリをファイルごとgitにアップロードすればできるのでしょうか

**Reply by ciffelia** [reply]
通常の使い方であればビルドは必要ありませんが、Koeに変更を加えてビルドしたいということでしょうか？

**Reply by maruo12** [reply]
ユーザーネームの読み上げ停止をしたくてビルドをするというディスカッションがあったのでしたいなぁと思ってます

**Reply by ciffelia** [reply]
ビルドするには、リポジトリをクローンしたディレクトリをターミナルで開いて`docker build -t koe-fork .`を実行してください。このページが参考になると思います。
https://penpen-dev.com/blog/docker-build-command/

**Reply by maruo12** [reply]
ようやくできました！ありがとうございます！　
ユーザーが入室したときなどにユーザーの名前を読み上げる機能などはありませんか？

**Reply by ciffelia** [reply]
ありません。入退室時にテキストチャットで教えてくれるBotと組み合わせれば実現できると思います。
